[BUG] `build` fails due to error: package org.apache.http.util does not exist

### What is the bug?

```shell
./gradlew clean build
=======================================
OpenSearch Build Hamster says Hello!
  Gradle Version        : 7.5.1
  OS Info               : Mac OS X 12.6 (aarch64)
  JDK Version           : 17 (Eclipse Temurin JDK)
  JAVA_HOME             : /Library/Java/JavaVirtualMachines/temurin-17.jdk/Contents/Home
  Random Testing Seed   : CB26AFDD2B0149E9
  In FIPS 140 mode      : false
=======================================

> Task :javadoc
/Users/lukas.vlcek/projects/lukas-vlcek/opensearch-plugin-template-java/src/main/java/org/opensearch/path/to/plugin/RenamePlugin.java:13: warning: no comment
public class RenamePlugin extends Plugin {
       ^
1 warning

> Task :compileTestJava FAILED
/Users/lukas.vlcek/projects/lukas-vlcek/opensearch-plugin-template-java/src/test/java/org/opensearch/path/to/plugin/RenamePluginIT.java:11: error: package org.apache.http.util does not exist
import org.apache.http.util.EntityUtils;
                           ^
/Users/lukas.vlcek/projects/lukas-vlcek/opensearch-plugin-template-java/src/test/java/org/opensearch/path/to/plugin/RenamePluginIT.java:34: error: cannot find symbol
        String body = EntityUtils.toString(response.getEntity());
                      ^
  symbol:   variable EntityUtils
  location: class RenamePluginIT
Note: /Users/lukas.vlcek/projects/lukas-vlcek/opensearch-plugin-template-java/src/test/java/org/opensearch/path/to/plugin/RenamePluginIT.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
2 errors

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileTestJava'.
> Compilation failed; see the compiler error output for details.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1s
13 actionable tasks: 12 executed, 1 up-to-date
```

FYI: I am pushing PR with a fix.

--- a/src/test/java/org/opensearch/path/to/plugin/RenamePluginIT.java
+++ b/src/test/java/org/opensearch/path/to/plugin/RenamePluginIT.java
@@ -8,13 +8,15 @@
 package org.opensearch.path.to.plugin;
 
 import com.carrotsearch.randomizedtesting.annotations.ThreadLeakScope;
-import org.apache.http.util.EntityUtils;
+import org.apache.hc.core5.http.ParseException;
+import org.apache.hc.core5.http.io.entity.EntityUtils;
 import org.opensearch.client.Request;
 import org.opensearch.client.Response;
 import org.opensearch.plugins.Plugin;
 import org.opensearch.test.OpenSearchIntegTestCase;
 
 import java.io.IOException;
+import java.nio.charset.StandardCharsets;
 import java.util.Collection;
 import java.util.Collections;
 
@@ -29,9 +31,9 @@ public class RenamePluginIT extends OpenSearchIntegTestCase {
         return Collections.singletonList(RenamePlugin.class);
     }
 
-    public void testPluginInstalled() throws IOException {
+    public void testPluginInstalled() throws IOException, ParseException {
         Response response = createRestClient().performRequest(new Request("GET", "/_cat/plugins"));
-        String body = EntityUtils.toString(response.getEntity());
+        String body = EntityUtils.toString(response.getEntity(), StandardCharsets.UTF_8);
 
         logger.info("response body: {}", body);
         assertThat(body, containsString("rename"));

